docs/partner-apps/bundleb2b/b2b-edition.md

```diff
@@ -1,25 +1,22 @@
-# The B2B Edition of BigCommerce
+# B2B Edition 
 
-BundleB2B (B3) adds business-to-business (B2B) functionality to the BigCommerce platform, allowing businesses to easily facilitate B2B operations online. B3 provides a comprehensive suite of key B2B features to improve the B2B self-service experience for BigCommerce store owners and their customers.
+The BigCommerce platform offers BundleB2B (B3) functionality which allows businesses to streamline online operations and improve experience for BigCommerce store owners and their customers.
```

Comment:
```suggestion
BigCommerce offers optional business-to-business (B2B) functionality with BundleB2B (B3), allowing businesses to streamline online operations and improve the experience for BigCommerce store owners and their B2B customers.
```

- We use the "B2B" abbreviation below without defining it, so I added that part back in.